build.sh

```
@@ -0,0 +1 @@
+docker run -d --name nginx-test -p 8080:80 nginx:latest
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)